### DISCUSSION TITLE
How the bootstrap's gray color palette(gray-100 to gray-900) made and what is the formula behind it?

### DISCUSSION THREAD

**Discussion by GNSubrahmanyam** [twbs/bootstrap#34296] [General]
In my custom bootstrap theme i want to change the gray palette. My base gray color is `888888`. Can anyone help me out to make my gray palette.

Bootstrap's gray color palette for reference
```scss
$gray-100: #f8f9fa !default;
$gray-200: #e9ecef !default;
$gray-300: #dee2e6 !default;
$gray-400: #ced4da !default;
$gray-500: #adb5bd !default;
$gray-600: #6c757d !default;
$gray-700: #495057 !default;
$gray-800: #343a40 !default;
$gray-900: #212529 !default;
```

**Comment by GNSubrahmanyam** [comment]
@mdo

**Reply by mdo** [reply]
Honestly don't remember... I think I was just mixing colors after picking an initial mid-gray. 😅

**Reply by GNSubrahmanyam** [reply]
Hahaha

**Comment by geminorum** [comment]
[Color Shades Generator](https://mdigi.tools/color-shades/)
